packages/vscode/languages/astro-language-configuration.json

```diff
@@ -16,13 +14,15 @@
     { "open": "'", "close": "'" },
     { "open": "\"", "close": "\"" },
     { "open": "`", "close": "`" },
-    { "open": "<!--", "close": "-->", "notIn": ["comment", "string"] },
+    { "open": "{/*", "close": " */}", "notIn": ["comment", "string"] },
+    { "open": "<!--", "close": " -->", "notIn": ["comment", "string"] },
```

Comment:
Is it intentional that the close has a leading space?

Reply:
Yes! For comments it makes for a nicer experience since people usually add a space at the end. This is actually a consistency fix, we already had the space for JS comments but not HTML ones